app/controllers/calculators_controller.rb

```diff
@@ -1,11 +1,19 @@
 # frozen_string_literal: true
 
 class CalculatorsController < ApplicationController
+  before_action :find_calculator, except: :index
+
   def index
     @calculators = Calculator.friendly.all
   end
 
-  def show
+  def show; end
+
+  def calculate; end
+
+  private
+
+  def find_calculator
     @calculator = Calculator.friendly.find(params[:slug])
   end
-end
```

Comment:
Pls fix like here - https://github.com/ita-social-projects/ZeroWaste/pull/163/files/13c6f7575e89807fac3fd1f6fdab3986907930bc#diff-afbfc0b5090a2faff6dfce6e4f9cbf43e5f92daa710ccc3b4a99daef3d2fa173R88

Reply:
The last line should be empty